documentation/api/v1/async_callback_api_doc.rb

```diff
@@ -0,0 +1,151 @@
+require 'swagger/blocks'
+
+module AsyncCallbackApiDoc
+  include Swagger::Blocks
+
+  UUID_DESC = 'The unique ID of the payload calling back'
+  UUID_EXAMPLE = '6dde5ce0e94c9f43'
+  TIMESTAMP_DESC = 'The Unix format timestamp when the asynchronous payload called back (in epoch)'
+  TIMESTAMP_EXAMPLE = '1536777407'
+  LISTENER_URI_DESC = 'The URL of the listener which received the callback'
+  LISTENER_URI_EXAMPLE = 'tcp://192.168.1.7:4444'
+  HOST_DESC = 'The IP address from which the callback was received'
+  HOST_EXAMPLE = '203.0.113.5'
+  PORT_DESC = 'The port from which the callback was received'
+  PORT_EXAMPLE = '443'
+  WORKSPACE_ID_DESC = 'The ID of the workspace this payload belongs to'
+  WORKSPACE_ID_EXAMPLE = 'default'
+
+# Swagger documentation for payloads model
+  swagger_schema :AsyncCallback do
```

Comment:
Shouldn't we have an entry for `:name` and `:description` in the payload model?

Reply:
This comment was based on us heading in direction of incorrectly modifying AsyncCallback to have fields from Payloads.